setup.py

```diff
@@ -0,0 +1,12 @@
+from setuptools import setup
+
+requirements = [
+      'gym',
+      'chainer',
+      'numpy',
+      'Pillow']
+
+setup(name='text_localization_environment',
+      version='0.0.1',
```

Comment:
Actually, I'd prefer the versioning to start with 0.1.0.

```suggestion
      version='0.1.0',
```

See https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase for reasoning/explanation.